src/coreclr/src/vm/genmeth.cpp

```diff
@@ -1705,4 +1717,19 @@ BOOL MethodDesc::SatisfiesMethodConstraints(TypeHandle thParent, BOOL fThrowIfNo
     return TRUE;
 }
 
+DWORD InstantiatedMethodDesc::GetDictionarySlotsSize()
+{
+    CONTRACTL
+    {
+        PRECONDITION(SystemDomain::SystemModule()->m_DictionaryCrst.OwnedByCurrentThread());
+    }
+    CONTRACTL_END
+
+    ULONG_PTR* pDictionarySlots = (ULONG_PTR*)IMD_GetMethodDictionary();
```

Comment:
This does not seem to be DACized correctly. 

You may want to move this logic into `class Dictionary`. There are other similar helpers already.

Reply:
But this is under !DACCESS_COMPILE. Shouldn't that be enough?

Reply:
You are right - it is not necessary to DACize for correctness.

But it would be still nice to have it on Dictionary like all other similar methods; and use  `TADDR` instead of `ULONG_PTR` since that's the type used to generic dictionary slots.

Reply:
Ok, I'll move these helpers to class Dictionary